src/modules/drawing/components/DrawingSelectInteraction.vue

```diff
@@ -90,7 +90,7 @@ export default {
             layers: [this.getDrawingLayer()],
             // As we've seen with the old viewer, some small features were hard
             // to select. We will try to add a bigger hit tolerance to mitigate that.
-            hitTolerance: 3,
+            hitTolerance: 6,
```

Comment:
as we are using this value in `DrawingTooltip.vue` too, it would maybe good to export it to `config.js` as a const

Reply:
I'll amend the config and check if a tolerance of 6 works with the modifyInteraction.